ZelBack/config/default.js

```diff
@@ -47,6 +47,7 @@ module.exports = {
         appsInformation: 'zelappsinformation', // stores actual state of flux app configuration info - initial state and its overwrites with update messages
         appsTemporaryMessages: 'zelappstemporarymessages', // storages for all flux apps messages that are not yet confirmed on the flux network
         appsLocations: 'zelappslocation', // stores location of flux apps as documents containing name, hash, ip, obtainedAt
+        appsInstallingLocations: 'appsInstallingLocations', // stores install location of flux apps as documents containing name, ip, obtainedAt
```

Comment:
currently we have a convention of having it all small letters. This is better for readability but just a note. I think in the future we want to make it in the camelcase all and get rid of the zel prefix